OxfordBodleian/Juel-Jensen/BDLaethf31.xml

```diff
@@ -138,47 +225,153 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            aids for identifying weekday readings</ref>. Erasures are sometimes marked with thin lines (e.g. on <locus target="#11rb"/>). 
+                                        Erasures are also found in the upper margins in correspondence of the incipit of the weekday readings (<locus target="#16vb #20va #40vb #49ra #62ra #69rb"/>).
+                                        The name of an individual, indicated as <foreign xml:lang="gez">ገበሩ፡</foreign> ‘his servant’ (probably either the owner, the patron or the donor of 
+                                        the manuscript), has been erased on <locus target="#3r"/>. Few words are washed out on <locus target="#46va #53vb #64vb"/>.
+                                    </desc>
+                                </item>
+                                <item xml:id="e5">
+                                    <desc>
+                                        A <foreign xml:lang="gez">tamallas</foreign>-sign in the shape of a cross indicates on <locus target="#49vb"/> the omission of a portion of text, 
+                                        which is written in the upper margin. Both the <foreign xml:lang="gez">tamallas</foreign>-sign and the omitted text are written in the same hand as 
+                                        that of <ref target="#e3">the aids for the weekday readings</ref> and of many corrections.
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                       <!-- <decoDesc>
                         </decoDesc>-->
                         
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
+                        <bindingDesc>
+                            <binding contemporary="true">
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards partially covered with reddish-brown <material key="leather"/>.
+                                </decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+                                    <material key="wood"/>       
+                                    <material key="leather"/>     
+                                </decoNote>
+                                <decoNote xml:id="b3" type="Cover" color="reddish-brown">The outer side of the front board, the spine and half of the outer side of the back board are 
+                                    covered with leather.</decoNote>
+                                <decoNote xml:id="b4" type="SewingStations">2, originally 4</decoNote>
```

Comment:
@PietroLiuzzo is that fine or could there be problems with text inside decoNote  type="SewingStations"?

Reply:
If inside `decoNote type="SewingStations"` we simply want to give the number of the currently used sewing stations (2 in the present case), I can just delete the exceeding text, which is btw somehow redundant since the same information is also given in a more discursive way under `<condition/>` .